refgram/syntax/index.html

```diff
@@ -672,16 +672,16 @@ <h4 id="object_incorporation_pattern"><a href="#object_incorporation_pattern">Ob
 <p>An incorporated CP is headed by a C carrying the <img class="tone" src="../assets/tones/d4.png"> tone:</p>
 
 <div class="gloss">
-	ma z+ao /suq /sa (YI) v~a bore (/PIANETE) ?<br/>
-	Ma zao súq sá chı ꝡâ bore píanete?<br/>
+	󱚰󱚺 󱚸󱚺󱛎󱛃 󱚺󱛊󱚲󱛂 󱚺󱛊󱚺 󱛘󱚿󱚹󱛙 󱛔 󱛁󱚺󱛌 󱚲󱛃󱚻󱚴 󱛘󱚳󱛊󱚹󱛍󱚺󱚵󱚴󱚷󱚴󱛙 󱛗<br/>
+	Ma zao súq sá chı, ꝡâ bore píanete?<br/>
 	whether familiar *2s* some believe *comp* flat the-planet<br/>
 	! 'Do you know someone who believes that the planet is flat?'
 </div>
 	
 <p>When incorporating a DP headed by the determiner <img class="tone" src="../assets/tones/d2.png">, its allomorph <span class="glyphs">lo</span> <span class="toaq">lo</span> must be used. This is because replacing the raw tonal morpheme <img class="tone" src="../assets/tones/d2.png"> with the tonal morpheme <img class="tone" src="../assets/tones/d4.png"> would result in forming an adverbial adjunct.</p>
```

Comment:
This **lo** was also left untranscribed:
```suggestion
<p>When incorporating a DP headed by the determiner <img class="tone" src="../assets/tones/d2.png">, its allomorph <span class="glyphs">󱚼󱛃</span> <span class="toaq">lo</span> must be used. This is because replacing the raw tonal morpheme <img class="tone" src="../assets/tones/d2.png"> with the tonal morpheme <img class="tone" src="../assets/tones/d4.png"> would result in forming an adverbial adjunct.</p>
```